README.md

```diff
@@ -273,6 +273,7 @@ export const unplugin = createUnplugin((options: UserOptions, meta) => {
 - [unplugin-sentry](https://github.com/kricsleo/unplugin-sentry)
 - [unplugin-imagemin](https://github.com/ErKeLost/unplugin-imagemin)
 - [unplugin-typedotenv](https://github.com/ssssota/typedotenv)
+- [unplugin-sentry-official](https://github.com/getsentry/sentry-javascript-bundler-plugins)
```

Comment:
Is the name `unplugin-sentry-official` a bit confusing? Maybe better to name it as `sentry-javascript-bundler-plugins`?

Reply:
I'm totally fine with that. I was just trying to follow the similar naming convention as the others :D